card/mod/01_core/spec/format/html_format_spec.rb

```diff
@@ -30,11 +30,12 @@
     end
 
     context "Cards with special views" do
-
     end
 
     context "Simple page with Default Layout" do
       before do
+        ca = Card.new name: 'ABook', type: 'Book'
+        rendered =  ca.format.render( :edit )
```

Comment:
Useless assignment to variable - `rendered`.<br>Operator `=` should be surrounded by a single space.<br>Space inside parentheses detected.